docs/using.adoc

```diff
@@ -188,8 +188,7 @@ NOTE: Testing has been done with Eclipse 4.5.0.
 [[using-the-box-via-ssh]]
 == Using the box via SSH
 
-Today, most users will work inside the Vagrant box. Access the box by
-using `ssh` to login to it with the following command:
+You can access the box by using `ssh` to login to it with the following command:
```

Comment:
@thatdocslady Could you update the phrase `vagrant box` to `vagrant VM` as in context of Vagrant `Box` and `VM` has different meaning. It is similar to ISO and running Virtual Machine.